crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -780,6 +791,11 @@ def raise_node_upgrade_error(message = "")
         raise ::Crowbar::Error::Upgrade::NodeError.new(message)
       end
 
+      def raise_live_migration_error(message = "")
+        Rails.logger.error(message)
+        raise ::Crowbar::Error::Upgrade::LiveMigrationError.new(message)
```

Comment:
Ah, sorry I missed this. I think it would. But maybe I did not pay attention that much